ui/src/components/EntitySearch/EntitySearchResults.scss

```diff
@@ -1,14 +1,19 @@
 @import "app/variables.scss";
 @import "app/mixins.scss";
 
+.EntitySearchResults-outer-container {
+  position: relative;
+}
+
+.EntitySearchResults-inner-container {
+  overflow-x: scroll;
+  overflow-y: visible;
+  padding: 0;
+  position: relative;
```

Comment:
@kjacks Curious whether you deliberately organise these to be alphabetically ordered, wether it's handled by a tool, or is it just luck?

Reply:
heheh nope just luck XD

In an ideal world I guess I would try to follow one of the CSS style guides like SMACSS or BEM to determine the order buuuut have never been quite sure if it's worth it...